libs/application/template-api-modules/src/lib/modules/templates/parental-leave/parental-leave.utils.ts

```diff
@@ -230,7 +230,7 @@ export const transformApplicationToParentalLeaveDTO = (
     expectedDateOfBirth: selectedChild.expectedDateOfBirth,
     // TODO: get true date of birth, not expected
     // will get it from a new Þjóðskrá API (returns children in custody of a national registry id)
-    dateOfBirth: selectedChild.expectedDateOfBirth,
+    dateOfBirth: '',
```

Comment:
Should this be undefined instead? Or does that break the validation?

Reply:
The C# API that receives this uses `isNullOrEmpty` so I decided to have it empty.